CollAction/Models/ProjectViewModels/ParticipatingInProjectsViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Extensions.Localization;
+
+namespace CollAction.Models.ProjectViewModels
+{
+    public class ParticipatingInProjectsViewModel : FindProjectsViewModel
```

Comment:
I wouldn't use inheritance in this case.

You're creating a situation where if someone changes something in FindProjects, it might break or change something in the user account page. Makes testing harder and creates a "subtype" in a case where they aren't really related.

Reply:
Good point. I added the property to the FindProjectsViewModel and made it nullable. Works better with the React component as well; SubscribedToEmail == true or false: show unsubscribe/subscribe button, if null then don't show button.